sendgrid-suppresions.md

```diff
@@ -0,0 +1,51 @@
+## Importing SendGrid Suppression List
+
+ SparkPost CLI can import your SendGrid suppression List(s). However, it has to be in specific format.
+ Follow along to learn how to export suppressions from SendGrid and format them that they're importable by SparkPost.   
+
+### Import Command
```

Comment:
That's what I did initially but it was un-DRY :P. To me, goto here is simpler that bigger block of snippet code which is identical (in our example) for each step.